stratum/hal/lib/common/common.proto

```diff
@@ -1171,6 +1171,11 @@ message FecStatus {
   FecMode mode = 1;
 }
 
+// Wrapper around the SDK port ID.
+message SdkPortId {
```

Comment:
```suggestion
message SdnPortIdOverride {
```

If the port translation is not performed by the target, return the port that should be used.